src/retrocookie/core.py

```diff
@@ -36,12 +36,17 @@ def find_template_directory(repository: git.Repository) -> Path:
 
 
 def load_context(repository: git.Repository, ref: str) -> Dict[str, Any]:
-    """Load the context from the .cookiecutter.json file."""
-    path = Path(".cookiecutter.json")
-    text = repository.read_text(path, ref=ref)
-    data = json.loads(text)
+    """Load the context from the .cookiecutter.json or .cruft.json file."""
+    try:
+        path = Path(".cookiecutter.json")
+        text = repository.read_text(path, ref=ref)
+        data = json.loads(text)
+    except KeyError:
```

Comment:
Why KeyError?

Reply:
Is that what `git.Repository.read_text` gives you when it doesn't find the file?

Reply:
It seems to be!
```
repository.read_text(path, ref=ref)
Traceback (most recent call last):
  File "/home/micael/projects/retrocookie/src/retrocookie/core.py", line 42, in load_context
    text = repository.read_text(path, ref=ref)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/micael/projects/retrocookie/src/retrocookie/git.py", line 200, in read_text
    blob = functools.reduce(operator.truediv, path.parts, commit.tree)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: '.cookiecutter.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/micael/projects/retrocookie/src/retrocookie/git.py", line 200, in read_text
    blob = functools.reduce(operator.truediv, path.parts, commit.tree)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: '.cookiecutter.json'
```

Reply:
Do you think it'd be better to, for example, modify the `read_text` method to raise a clearer error, like a [`FileNotFoundError`](https://docs.python.org/3/library/exceptions.html#FileNotFoundError), instead?

Reply:
Possibly, but not in this PR. Sorry for the slow review, have little bandwidth atm

Reply:
No worries. And sorry for pinging you. I had forgotten about this PR, then realized it had been over a year with no activity.